packages/@aws-cdk/aws-ec2/lib/peer.ts

```diff
@@ -0,0 +1,166 @@
+import { Connections, IConnectable } from "./connections";
+
+/**
+ * Interface for classes that provide the peer-specification parts of a security group rule
+ */
+export interface IPeer extends IConnectable {
+  /**
+   * Whether the rule can be inlined into a SecurityGroup or not
+   */
+  readonly canInlineRule: boolean;
+
+  /**
+   * A unique identifier for this connection peer
+   */
+  readonly uniqueId: string;
```

Comment:
Would it make sense to make `IPeer` an `IConstruct` and then you can use `node.uniqueId`?

Reply:
Not all peers are constructs.